website/source/docs/agent/checks.html.md

```diff
@@ -42,25 +42,27 @@ There are several different kinds of checks:
   blog post](https://www.hashicorp.com/blog/protecting-consul-from-rce-risk-in-specific-configurations)
   for more details.
 
-* HTTP + Interval - These checks make an HTTP `GET` request every Interval (e.g.
-  every 30 seconds) to the specified URL. The status of the service depends on
-  the HTTP response code: any `2xx` code is considered passing, a `429 Too Many
-  Requests` is a warning, and anything else is a failure. This type of check
+* HTTP + Interval - These checks make an HTTP `GET` request to the specified URL,
+  waiting the specified Interval amount of time (eg. 30 seconds) between requests.
```

Comment:
I think this reads a little easier if the `eg` isn't in the middle of the sentence:
```suggestion
  waiting between requests for the specified interval time (eg. 30 seconds).
```

Also we probably shouldn't be capitalizing "interval" in the description anymore.

Reply:
sounds good! shall I codify interval like `timeout` is? (`interval`)? that's sort of what i was expecting in the docs